content/en/blog/2022-12-01-november-update/index.md

```diff
@@ -0,0 +1,379 @@
+  variable substitutions from Secrets and ConfigMaps are skipped, and no
+  connection to the cluster is made.
+- Use signed OCI Helm chart for
+  [kube-prometheus-stack](/flux/guides/monitoring/).
+
+Check out these new pieces of documentation:
+
+- Guide: [AWS CodeCommit bootstrap](/flux/use-cases/aws-codecommit/)
+- Guide: [Azure DevOps
+  bootstrap](/flux/use-cases/azure/#flux-installation-for-azure-devops)
+
+💖 Big thanks to all the Flux contributors that helped us with this
+release!
+
+### Flux Roadmap Updates
+
+Roadmap - we are rushing forward towards GA! <https://fluxcd.io/roadmap>
+
+Starting with release v0.37, we started solidifying all required changes
+for the Bootstrap GA milestone targeted to Q1 2023. That release should
+include all major changes from a Git perspective that we want to ship
+for GA. Please make sure you upgrade as soon as possible and provide us
+with feedback, so we can work on it before the GA release.
+
+Upcoming in the next release is a new feature for Image Automation
+Controller: `GitShallowClones`. You can already check it out in the
+recently published release candidate. If you are interested, you can
+reach out via the PR or on Slack:
+<https://github.com/fluxcd/image-automation-controller/pull/463>
+
+### Security news
+
+<https://fluxcd.io/blog/2022/11/verify-the-integrity-of-the-helm-charts-stored-as-oci-artifacts-before-reconciling-them-with-flux/>
+
+Flux has now an AppArmor profile distributed via the Kubernetes-SIG/security-profiles-operator project:
+<https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/examples/apparmorprofile-flux-controllers.yaml>
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+The team upgraded Flux Subsystem for Argo aka Flamingo to support
+Flux v0.37 and Argo CD v2.5.3, v2.4.17, v2.3.11 and v2.2.16.
+
+#### Terraform-controller
+
+The team has released Weave TF-controller v0.13.1 and recently updated
+its Helm chart to v0.9.3.  In this version, the team started shipping the
+AWS Package for TF-controller. The AWS Packge is an OCI Image which
+contains a set of Terraform primitive modules that you can use
+out-of-the-box to provision your Terraform resources by describing them as
+YAML.Please visit the package repository for more information:
+<https://github.com/tf-controller/aws-primitive-modules>.
+
+#### Weave GitOps
+
+<https://github.com/weaveworks/weave-gitops/releases/tag/v0.10.2>
+<https://github.com/weaveworks/weave-gitops/releases/tag/v0.11.0>
+
```

Comment:
@jamwils If you, or anyone in your team, could help out with this section, that'd be great!

Reply:
Ping. If possible, I would like to release this update today.